packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerHighlight/PdfViewerHighlight.tsx

```diff
@@ -127,4 +162,33 @@ const useHighlighter = ({
   }, [document, isReady, pageNum, pdfRenderedText, processedDoc, textMappings]);
 };
 
+function useScrollIntoActiveHighlight(
+  highlightDivRef: React.MutableRefObject<HTMLDivElement | null>,
+  shapes: (HighlightShape | undefined)[],
```

Comment:
`shapes` can be an array of `undefined`?

Reply:
no. I fixed the code